include/highfive/H5DataType.hpp

```diff
@@ -136,6 +136,37 @@ class CompoundType : public DataType {
     void create(size_t size = 0);
 };
 
+template <typename T>
+class EnumType : public DataType {
+public:
+    struct member_def {
```

Comment:
```suggestion
    struct Member {
```
? Reads easier…

Reply:
yeah but it would break from the naming used for `CompoundType`

Reply:
fine